.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+sudo: required
+
+services:
+  - docker
+go:
+  - 1.9
+  - "1.10"
+
+script:
+  - make test
```

Comment:
can we add _make fmt_ as well.

Reply:
`make test` includes `fmt` test as well. It is part of Makefile.